README.md

```diff
@@ -169,6 +170,21 @@ This option is used by JetBrains Marketplace by default.
   * `no`: all internal API usages will be reported. This is the default value.
   * `jetbrains-plugins`: internal API usages by JetBrains plugins will not be reported.  
 
+* `-mute` will mute (ignore) a specified plugin problem.
```

Comment:
Maybe let's change it to `ignore`?

I just thought that:
- mute - rule is executed, but its result is ignored
- ignore - rule is not executed at all

Which case is that?

Also, please stick to a single term: mute or ignore. I can see both used interchangeably in other places. It may be confusing.

Reply:
A problem is a result of a rule. However, PV does not have a strict concept of a rule.
Rules are always executed, but their results are ignored. The reasons may be many, and some of those are already in place due to other mechanisms.

Hence, _mute_.

In CLI switch, it's `-mute`. In other places, it is consistently _ignore_, unless there has been an accidental. omission